doc/examples/animations-pulsating-sphere.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Animations of a Pulsating Sphere"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sfs\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import animations_pulsating_sphere as animation\n",
```

Comment:
Is it possible to make a link that works in sphinx as well as in jupyter notebook?

Reply:
Yes it is. But it's not possible to use inline formatting within a link in Sphinx (or more exactly, in reStructuredText, which is currently an intermediate format used in `nbsphinx`), see http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#gotchas.

IOW, instead of

    [`animations_pulsating_sphere.py`](animations_pulsating_sphere.py)

... you'll have to use

    [animations_pulsating_sphere.py](animations_pulsating_sphere.py)

... which isn't that bad IMHO.